share/spack/docker/spack_ubuntu/handle-ssh.sh

```diff
@@ -0,0 +1,24 @@
+uid="`id -u`"
```

Comment:
isn't this file needed in fedora as well?

Reply:
Right.  Fedora et al is coming, soon (TM),